checker/jdk/determinism/src/java/util/Objects.java

```diff
@@ -140,7 +140,7 @@ else if (a == null || b == null)
      * @see Object#toString
      * @see String#valueOf(Object)
      */
-    public static @PolyDet String toString(@PolyDet Object o) {
+    public static @PolyDet("up") String toString(@PolyDet Object o) {
```

Comment:
Since we're changing `toString` to `@NonDet` in `Object`, the same should apply here and this return type should be `@NonDet`